src/containers/widget/index.js

```diff
@@ -16,28 +16,42 @@ import { WidgetWrapper } from "../../utils/widget-wrapper";
 
 const DEFAULT_API_BASE_URL = "https://area402.herokuapp.com";
 
+function getBasicState(props) {
+  const {
+    apiToken,
+    currency,
+    showModal,
+    apiBaseUrl,
+    widgetTitle,
+    paymentOptions,
+    welcomeMessage,
+    enableEmailSubscription,
+  } = props;
+
+  return {
+    apiToken: apiToken || "",
+    currency: currency || "€",
+    apiBaseUrl: apiBaseUrl || "",
+    isModalOpen: showModal || false,
+    widgetTitle: widgetTitle || "",
+    welcomeMessage: welcomeMessage || "",
+    paymentOptions: paymentOptions || [],
```

Comment:
paymentOptions are now an object, see defaults.

Reply:
Will change it